src/main/scala/chisel3/package.scala

```diff
@@ -1,515 +0,0 @@
-        val reg = if (init ne null) {
-          RegInit(t, init)
-        } else {
-          chisel3.core.Reg(t)
-        }
-        if (next ne null) {
-          reg := next
-        }
-        reg
-      } else if (next ne null) {
-        if (init ne null) {
-          RegNext(next, init)
-        } else {
-          RegNext(next)
-        }
-      } else if (init ne null) {
-        RegInit(init)
-      } else {
-        throwException("cannot infer type")
-      }
-    }
-  }
-
-  val when = chisel3.core.when
-  type WhenContext = chisel3.core.WhenContext
-
-  type Printable = chisel3.core.Printable
-  val Printable = chisel3.core.Printable
-  type Printables = chisel3.core.Printables
-  val Printables = chisel3.core.Printables
-  type PString = chisel3.core.PString
-  val PString = chisel3.core.PString
-  type FirrtlFormat = chisel3.core.FirrtlFormat
-  val FirrtlFormat = chisel3.core.FirrtlFormat
-  type Decimal = chisel3.core.Decimal
-  val Decimal = chisel3.core.Decimal
-  type Hexadecimal = chisel3.core.Hexadecimal
-  val Hexadecimal = chisel3.core.Hexadecimal
-  type Binary = chisel3.core.Binary
-  val Binary = chisel3.core.Binary
-  type Character = chisel3.core.Character
-  val Character = chisel3.core.Character
-  type Name = chisel3.core.Name
-  val Name = chisel3.core.Name
-  type FullName = chisel3.core.FullName
-  val FullName = chisel3.core.FullName
-  val Percent = chisel3.core.Percent
-
-  /** Implicit for custom Printable string interpolator */
-  implicit class PrintableHelper(val sc: StringContext) extends AnyVal {
-    /** Custom string interpolator for generating Printables: p"..."
-      * Will call .toString on any non-Printable arguments (mimicking s"...")
-      */
-    def p(args: Any*): Printable = {
-      sc.checkLengths(args) // Enforce sc.parts.size == pargs.size + 1
-      val pargs: Seq[Option[Printable]] = args map {
-        case p: Printable => Some(p)
-        case d: Data => Some(d.toPrintable)
-        case any => for {
-          v <- Option(any) // Handle null inputs
-          str = v.toString
-          if !str.isEmpty // Handle empty Strings
-        } yield PString(str)
-      }
-      val parts = sc.parts map StringContext.treatEscapes
-      // Zip sc.parts and pargs together ito flat Seq
-      // eg. Seq(sc.parts(0), pargs(0), sc.parts(1), pargs(1), ...)
-      val seq = for { // append None because sc.parts.size == pargs.size + 1
-        (literal, arg) <- parts zip (pargs :+ None)
-        optPable <- Seq(Some(PString(literal)), arg)
-        pable <- optPable // Remove Option[_]
-      } yield pable
-      Printables(seq)
-    }
-  }
-
-  implicit def string2Printable(str: String): Printable = PString(str)
-
-  implicit class fromBigIntToLiteral(x: BigInt) extends chisel3.core.fromBigIntToLiteral(x)
-  implicit class fromtIntToLiteral(x: Int) extends chisel3.core.fromIntToLiteral(x)
-  implicit class fromtLongToLiteral(x: Long) extends chisel3.core.fromLongToLiteral(x)
```

Comment:
This resolves the burning question of why from**t**IntToLiteral is written that way